src/main/scala/org/ergoplatform/nodeView/state/ErgoStateContext.scala

```diff
@@ -254,12 +253,11 @@ object ErgoStateContext {
   }
 
   /**
-    * Initialize empty state context with fake PreHeader
+    * Initialize empty state context
     */
   def empty(genesisStateDigest: ADDigest, settings: ErgoSettings): ErgoStateContext = {
-    new ErgoStateContext(Seq.empty, None, genesisStateDigest, LaunchParameters, ErgoValidationSettings.initial, VotingData.empty)(settings.chainSettings.voting)
-      .upcoming(org.ergoplatform.mining.group.generator, 0L, settings.chainSettings.initialNBits, Array.fill(3)(0.toByte),
-        ErgoValidationSettingsUpdate.empty, 0.toByte)
+    new ErgoStateContext(Seq.empty, None, genesisStateDigest, LaunchParameters, ErgoValidationSettings.initial,
```

Comment:
@aslesarenko It's not. I described why I did this in PR's description:
>made ErgoStateContext.empty to return ErgoStateContext rather than UpcomingStateContext since it later ends up in ErgoStateContextSerializer.serialize and parse does not support it thus PreHeader was lost;